### ISSUE TITLE
Collabora Office: clicking a "Recent file" then immediately maximising or windowing crashes the app (Chrome OS)

### ISSUE THREAD

**Issue by rpear** [CollaboraOnline/online#3500]
**Describe the bug**
On a Chromebook, clicking a "Recent file" then immediately maximising or windowing the app while the file is loading crashes the app. Minimising the app does not trigger the crash, it's fine.

I can't make it crash in Collabora **Online** 6.4.13 if maximising, windowing or minimising the browser whilst opening a file.

**Symptoms**
It hangs on "loading..." for ~10 seconds before the OS says "Collabora Office isn't responding", Close, Wait, or Send Feedback.

**Workaround**
Wait until the document is loaded before maximising or windowing the app.

**Versions**
Chrome OS latest version Version 94.0.4606.104 (Official Build) (64-bit), I believe it misbehaved on earlier versions too.
App: Collabora Office 6.4.13 (online git-hash: cea7112 core git-hash: 70f652274508)

**Comment by thebearon** [comment]
Indeed, same happens when you double-tap a document in the recent documents list.

**Comment by thebearon** [comment]
Ah right, that one's reported separately in #3497.

**Comment by rpear** [comment]
This is no longer a problem for me in version: 21.11.3.6 of the Chromebook app, online git-hash: eb73aa348, core git-hash: e90143624dba, it was only a problem for me in 6.4.

**Comment by pedropintosilva** [comment]
Thanks @rpear for reporting and updating it ! Much appreciated

**Comment by rpear** [comment]
Closing

**Comment by rpear** [comment]
Reproduced the crash in 23.05.5.2

**Comment by rpear** [comment]
This is OK now again, no more crashes with 24.04.12.4

It is OK double-tapping a document in the recent documents list.
It is OK clicking a "Recent file" then immediately maximising or windowing.

Maybe it is because I am using a relatively super powerful Chromebook at the moment.  It could have been a ChromeOS issue, Google made big changes to the way Android runs on Chromebooks.
